starter-code/src/array-functions.js

```diff
@@ -1,33 +1,51 @@
-var ArrayFunctions = function (){}
+// var ArrayFunctions = function (){};
+class ArrayFunctions {
+  
+  cosntructor () {
+  }
+
```

Comment:
Because I didn't have to create an instance of the class ArrayFunctions to run those methods I define them as statics then they can be called from the class itself.